spec/models/aged_offspring_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.describe AgedOffspring, type: :model do
+  active_record_offspring = 'activerecord.errors.models.offspring.attributes'
+
+  context "when authenticated as admin" do
+    let(:admin) { FactoryGirl.create(:admin) }
+
+    after(:each) do
+      Warden.test_reset!
+    end
+
+    describe "validating attribute first_name " do
+      it "is invalid without first_name" do
+        off = FactoryGirl.build(:agedOffspring, first_name: "")
+        off.valid?
+        expect(off.errors[:first_name]).to include(I18n.t('f_name.blank', scope: active_record_offspring))
+        off.first_name = "Peter"
```

Comment:
You should test the factory and not every case, and then you haven't done it at the end